boards/arm/nrf9160_pca10090/nrf9160_pca10090ns_defconfig

```diff
@@ -0,0 +1,26 @@
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_NRF=y
+CONFIG_SOC_SERIES_NRF91X=y
+CONFIG_SOC_NRF9160_SICA=y
+CONFIG_BOARD_NRF9160_PCA10090=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable TrustZone-M
+CONFIG_ARM_TRUSTZONE_M=y
+
+# This Board implies building Non-Secure firmware
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
```

Comment:
This one is in accordance with v2m_musca, where the "non-secure" version of the board determines that we are building a NONSECURE firmware.